client/components/domains/search-filters/dropdown-filters.jsx

```diff
@@ -231,7 +231,7 @@ export class DropdownFilters extends Component {
 				{ showTldFilter && (
 					<ValidationFieldset className="search-filters__tld-filters">
 						<FormLabel className="search-filters__label" htmlFor="search-filters-max-characters">
-							{ translate( 'Extensions' ) }:
+							{ translate( 'Extensions', { context: 'domain extension, like .com' } ) }:
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **69** times:
`translate( 'Extensions' )` *ES Score: 11*